curriculum/challenges/english/20-upcoming-python/learn-string-manipulation-by-building-a-cipher/65549561463f0016876e852c.md

```diff
@@ -0,0 +1,62 @@
+---
+id: 65549561463f0016876e852c
+title: Step 55
+challengeType: 20
+dashedName: step-55
+---
+
+# --description--
+
+Because your key is shorter than the text to encrypt, you will need to repeat it to generate the whole encrypted text.
```

Comment:
```suggestion
Since your key is shorter than the text that you will have to encrypt, you will need to repeat it to generate the whole encrypted text.
```